vars/RedHat.yml

```diff
@@ -18,3 +18,11 @@ postgresql_unix_socket_directories:
 
 postgresql_fdw_mysql_packages: "mysql_fdw_{{ postgresql_version_terse }}"
 postgresql_fdw_ogr_packages: "ogr_fdw{{ postgresql_version_terse }}"
+
+postgresql_packages:
+  - ca-certificates
+  - python-psycopg2
+  - python-pycurl
+  - glibc-common
+  - epel-release
```

Comment:
I think, the setup of packages must be done in to steps:
1. Enable EPEL repository
2. Install the packages, using EPEL repository

Reply:
@genben the way I understand it, Just the postgresql packages, like `postgresql12-server`, `postgesl112`, and `postgresql12-contrib` require the EPEL repository, those packages are installed on a later step.  

The only change here was moving the list of dependencies to a variable
```yaml
      - name: PostgreSQL | Install all the required dependencies | yum
        yum:
          name: ["ca-certificates","python-psycopg2", "python-pycurl", "glibc-common","epel-release","libselinux-python"]
          state: present
```

```
      - name: PostgreSQL | Install all the required dependencies (yum) | RHEL
        yum:
          name: "{{ postgresql_packages }}"
          state: present
          update_cache: yes
```

A more descriptive name for the variables would be  `postgresql_packages_dependencies`, but i was trying to follow the upstream repo